src/System.Windows.Forms/src/System/Windows/Forms/ControlState.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Windows.Forms
+{
+    public partial class Control
+    {
+        [Flags]
+        private protected enum ControlState
```

Comment:
As per the [naming guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/enum) flag enums should be pluralised
![image](https://user-images.githubusercontent.com/4403806/62515880-3a6ffb80-b82c-11e9-841e-8b4fb895611a.png)

```suggestion
        private protected enum ControlStates
```

Reply:
Further thinking about the name - the enum is in the context of `Control`, are we gaining much by having `Control` in the name of the enum - `Control.ControlStates.Xyz`?
Can we shorten the name to:
```suggestion
        private protected enum States
```